### ISSUE TITLE
Graphs are not displayed

### ISSUE THREAD

**Issue by densenator** [Cacti/cacti#4942]
Maybe After update spine to latest version, my graphs are not displayed.
![image](https://user-images.githubusercontent.com/18443709/194049259-7c189243-693d-49e8-9fd3-68c537bbcce3.png)
Log:
05.Oct.2022 14:16:47 - PCOMMAND Device NOTE: Recache Event Detected for Device

SPINE: Poller[[Main Poller](https://cacti.satgate.net/pollers.php?action=edit&id=1)] PID[2852065] PT[139630544393984] Device HT[1] DQ[[SNMP - Interface Statistics](https://cacti.satgate.net/data_queries.php?action=edit&id=4)] RECACHE ASSERT FAILED: '2630130<'

This is for all devices.

**Comment by densenator** [comment]
SPINE 1.2.23 last from branch 1.2.x

**Comment by bmfmancini** [comment]
All of your devices are showing recache?
What is your reindex method set to?

**Comment by densenator** [comment]
Yes, and i see only advsnced ping graph.
SNMP Uptime method

**Comment by TheWitness** [comment]
It's almost as if you lost SNMP access to the devices.

**Comment by densenator** [comment]
After i installed old version of spine is bacward to all right.
How help you to fix this issue?

**Comment by TheWitness** [comment]
Your "After i installed ..." statement make no sense.  I'll let @bmfmancini continue to work with you on this.

**Comment by densenator** [comment]
SPINE: Poller[[Main Poller](https://cacti.satgate.net/pollers.php?action=edit&id=1)] PID[2852065] PT[139630544393984] Device HT[1] DQ[[SNMP - Interface Statistics](https://cacti.satgate.net/data_queries.php?action=edit&id=4)] RECACHE ASSERT FAILED: '2630130<'
Maybe spine trying to find "<" symbol?

**Comment by TheWitness** [comment]
No, that error means that the value was not returned by the SNMP process.  It is the `poll_result` variable from the SNMP device and the inquiry to the sysUptimeInstance variable.

For the device in question, run the following two snmpget's using the devices snmp community, version, etc.

Classic SNMP sysUptimeInstance
`.1.3.6.1.2.1.1.3.0`

New Form SNMP sysUptimeInstance
`.1.3.6.1.6.3.10.2.1.3.0`

Then, run the following for us and return the results:

```sh
./spine -R -V 5 -S -f device_id -l device_id
```

Replace `device_id` with your devices device id.

**Comment by TheWitness** [comment]
Moving this to a Spine issue.
